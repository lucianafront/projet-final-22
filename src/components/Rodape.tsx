import React from "react";

function Rodape() {
  return (
    <div className="footer">
      <p>â’¸ Desenvolvido por Luciana S. Felix</p>
    </div>
  );
}

export default Rodape;