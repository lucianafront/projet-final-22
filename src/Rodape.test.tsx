import React from "react";
import { render } from "@testing-library/react";
import Rodape from "./components/Rodape";

describe("Rodape component", () => {
  test("renders Rodape component", () => {
    const { getByText } = render(<Rodape />);
    const footerElement = getByText("â’¸ Desenvolvido por Luciana S. Felix");
    expect(footerElement).toBeInTheDocument();
  });
  
  test("matches snapshot", () => {
    const { asFragment } = render(<Rodape />);
    expect(asFragment()).toMatchSnapshot();
  });
});