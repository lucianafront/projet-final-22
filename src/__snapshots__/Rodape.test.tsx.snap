// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rodape component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p>
      â’¸ Desenvolvido por Luciana S. Felix
    </p>
  </div>
</DocumentFragment>
`;
